packages/editor-ui/src/views/NodeView.vue

```diff
@@ -504,14 +504,14 @@ export default defineComponent({
 			// When entering this tab:
 			if (currentTab === MAIN_HEADER_TABS.WORKFLOW || isOpeningTemplate) {
 				if (workflowChanged || nodeViewNotInitialized || isOpeningTemplate) {
-					this.startLoading();
+					this.canvasStore.startLoading();
```

Comment:
Why not use loadingService composable directly?

Reply:
Because `loadingService` has state encapsulated only within the component but here we need to have it global for the canvas so we can control the loading state outside of NodeView component(for example in `src/components/MainHeader/WorkflowDetails.vue#614`)